legend-engine-ide-lsp-default-extensions/src/main/java/org/finos/legend/engine/ide/lsp/extension/core/FunctionExecutionSupport.java

```diff
@@ -124,6 +121,45 @@ default String getExecutionKey(PackageableElement element, Map<String, Object> a
         return "";
     }
 
+    static Iterable<? extends LegendExecutionResult> execute(AbstractLSPGrammarExtension extension, SectionState section, String entityPath, String commandId, Map<String, String> executableArgs, Map<String, Object> inputParameters)
```

Comment:
I would make extension of type FunctionExecutionSupport.  This APIs are for extensions that support function execution.

Reply:
Done